src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/GeneralType.php

```diff
@@ -46,25 +46,78 @@ class GeneralType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('catalog_mode', SwitchType::class)
-            ->add('catalog_mode_with_prices', SwitchType::class)
+            ->add('catalog_mode', SwitchType::class, [
+                'label' => $this->trans('Catalog mode', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans(
+                        'Catalog mode disables the shopping cart on your store. Visitors will be able to browse your products catalog, but not buy them.',
+                        'Admin.Shopparameters.Help'
+                    ) . '<br />' . $this->trans(
+                        'Have specific needs? Edit particular groups to let them see prices or not.',
+                        'Admin.Shopparameters.Help'
+                    ),
+            ])
+            ->add('catalog_mode_with_prices', SwitchType::class, [
+                'label' => $this->trans('Show prices', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans(
+                    'Display product prices when in catalog mode.',
+                    'Admin.Shopparameters.Help'
+                ),
+            ])
             ->add('new_days_number', IntegerType::class, [
+                'label' => $this->trans(
+                    'Number of days for which the product is considered \'new\'',
+                    'Admin.Shopparameters.Feature'
+                ),
                 'required' => false,
             ])
             ->add('short_description_limit', TextWithUnitType::class, [
+                'label' => $this->trans(
+                    'Max size of product summary',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'Set the maximum size of the summary of your product description (in characters).',
```

Comment:
I think we can remove this tooltip. The label is already clear enough and it's repetitive. + It's not clear: are we talking about the product summary or the description?